CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+## Coding style
+
+* HOL source code
+  - Spaces, no tabs
+  - No unicode
+  - `snake_case` (e.g. `bir_number_of_mem_splits_def`)
+
+
+
+## Branch policy
+
+### tags
+
+tags should have as many **completed features** as possible:
+ - no cheat
+ - must correctly compile
+ - self tests must succeed
+ - code should be tested
+
+Follow these instructions whenever you merge to master:
+  - `grep` for "cheat"
+  - check that the `README` is up to date (especially tool status)
+  - find a reviewer for your Pull Request
+
+### `master` branch
+
+`master` is the branch where every feature is available, but not necessarily finalized:
+  - Can cheat, but has to be avoided
+  - Code should not be be commented out
+  - **Holmake must work**, which is checked by our Travis CI
```

Comment:
Should we add that all tests must pass? That eventually shouldn't be the case, but we cannot really define failing tests for the time being. An alternative would be to always manually check the failing tests in Travis.

Reply:
good point, let's just keep it in mind for now and see how it goes